build/assets/ios/Showcase/Application/Core/Foundations/Assets/AssetsModel+Sections.swift

```diff
@@ -0,0 +1,356 @@
+//
+//  Vitamin iOS
+//  Apache License 2.0
+//
+
+import Foundation
+import VitaminCore
+
+enum AssetsModel {
+    struct Section: Identifiable {
+        var id: String {
+            name
+        }
+        let name: String
+        let items: [VitaminAsset]
+    }
+
+    static let sections = [
+        Assets.Section(name: "Flags", items: [
```

Comment:
I thinks there is an issue here it should be `AssetsModel` instead of `Assets `

Reply:
You're right, done here: https://github.com/Decathlon/vitamin-design/pull/232/commits/bad5ac9b2a218a7fe05e5d8677656a14eb2f4d7b